src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommand.java

```diff
@@ -199,6 +199,20 @@ private void setDryRun(boolean dryRun) {
     argumentsBuilder.setDryRun(dryRun);
   }
 
+  @Option(
+      names = "--rows-per-chunk",
+      defaultValue = "0",
+      description = {
+        "If larger than 0, the tool will attempt to use a chunked processing mode for scripts that"
+            + " support this, where the results for a supporting script are saved in chunks; this"
+            + " number defines the maximum rows per chunk holds. Currently, only querylogs.sql"
```

Comment:
I'd rather exclude listing of scripts with feature support – it's really easy to forget to update this part, as it's not a part of the documentation

Reply:
To clarify, do you suggest just removing the "Currently, only querylogs.sql..." part?

Reply:
Yes